src/Tizen.Uix.Tts/Tizen.Uix.Tts/SynthesizedPcmEventArgs.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+* Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+using System.ComponentModel;
+
+namespace Tizen.Uix.Tts
+{
+    /// <summary>
+    /// This class holds information related to the TTS SynthesizedPcm event.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class SynthesizedPcmEventArgs
```

Comment:
It should inherit from System.EventArgs

Reply:
Thank you for checking. I think other event args in Tts do not inherit the System.EventAgrs.
So Is it okay to fix it all other classes without ACR?